keyboards/planck/keymaps/navi/keymap.c

```diff
@@ -9,13 +9,12 @@
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
  * GNU General Public License for more details.
- * cd /c/qmk_firmware
- * make planck/rev4:navi
+ * cd /c/qmk_firmware-master... make planck/rev6:navi
  * You should have received a copy of the GNU General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-#pragma message "You may need to add LAYOUT_planck_grid to your keymap layers - see default for an example"
+#include QMK_KEYBOARD_H
 #include "planck.h"
```

Comment:
Lines 18-23 can be deleted; the `QMK_KEYBOARD_H` include on Line 17 includes all those files automatically.